### ISSUE TITLE
Help people find the published guide if they visit github directly

### ISSUE THREAD

**Issue by gcharters** [OpenLiberty/guides-common#134]
I noticed from the github traffic info that people are directly finding our guide repos from Google searches.  There's no indication in the guide repos that they shouldn't read the github rendered README.adoc files in the repos because they suck in their semi-raw form.  

We should include some text at the top of the README.adoc that points people to the built guide on openliberty.io.  This text should appear in the github rendered form when viewed in the repo, but be hidden or excluded by the build so it doesn't appear in the guide on openliberty.io

The repos having good description and referral info would also help here.  For example, the MP Rest Client repo just says this at the top:  "No description, website, or topics provided. ". A good example is, https://github.com/OpenLiberty/guide-rest-intro .

**Comment by yeekangc** [comment]
@evelinec will take the lead on this.

As noted, it may involve changes to building the guides for openliberty.io.  Tagging @kinueng @mbroz2  @megmulsy @cajaygle.

**Comment by mbroz2** [comment]
2 Action Items:
1) Ensure all guides have a github description & link to the guide on openliberty.io
2) Add text to the top of the README.adoc that directs users to openliberty.io; ensure this text does not appear in the guide on openliberty.io  (the work will be done here: https://github.com/OpenLiberty/openliberty.io/issues/346)

**Comment by steven1046** [comment]
If you specify this in the guide's asciidoc:
:repo-description: Visit the https://openliberty.io/guides/{projectid}.html[website] for the rendered version of the guide.
= Injecting dependencies into microservices

Learn how to use Contexts and Dependency Injection to manage and inject dependencies into microservices.

[.hidden]
{repo-description}

Asciidoc will automatically hide it on the guide on the website but it will show up in the Github README.

This way, it can be done the same way for all guides and also update the template to have this format so new guides will have it as well.

No code changes to the website are required for this approach.

**Comment by evelinec** [comment]
Action item 1 was completed. Working on item 2.

**Comment by evelinec** [comment]
Item 2 is prototyped with the cdi-intro guide via https://github.com/OpenLiberty/guide-cdi-intro/pull/45. @gcharters @yeekangc and @sishida, pls review. Thanks.

**Comment by evelinec** [comment]
@gcharters can you take a look at this guide's repo: https://github.com/OpenLiberty/guide-cdi-intro for the fixes in the repo description as well as the text added in the repo. The text in the repo should not be built in the website. If both are good, can we close this issue? Thanks.

**Comment by yeekangc** [comment]
@evelinec, I believe we will fix for all repos, correct?  I think we close this issue only after we fix everything.

**Comment by evelinec** [comment]
Yes @yeekangc. That's work in progress. Just posted one repo for Graham to take a look.
